td-reproducible-tool/readme.md

```diff
@@ -0,0 +1,61 @@
+# td-reproducible-tool
+
+## Background
+
+Elements break reproducible build (x86_64-unknown-uefi + target x86_64-unknown-none) <br>
+ <br>
+```
+1. Debug related sections (PE + ELF) 
+2. Code base path like \home\USERNAME\rust-td-payload\xxx\xxx.rs (PE + ELF) 
+3. CARGO_HOME like \home\USERNAME\.cargo\xxx\xxx.rs (PE + ELF) 
+4. RUSTUP_HOME like \home\USERNAME\.rustup\xxx\xxx.rs (PE + ELF) 
+5. TimeDateStamp field in COFF file header (PE only) 
+```
+
+Solution: <br>
+
+```
+1. Use strip = "symbols" option to [profile.release] 

```

Comment:
Please update to debuginfo

Reply:
> Please update to debuginfo

Still using symbols in the latest code change.